docs/guides/howto_behind_proxy.md

```diff
@@ -28,6 +28,14 @@ qdt --proxy-http "http://user:password@proxyserver.intra:8765"
 - it avoids potential conflict with "classic" proxy settings
 - it allows to use a specific network proxy for QDT (can be useful for some well controlled systems)
 
+### Use PAC file
+
+[PAC file](https://developer.mozilla.org/en-US/docs/Web/HTTP/Proxy_servers_and_tunneling/Proxy_Auto-Configuration_PAC_file) can be used by SysAdmin to define proxy with a set of rules depending on the url.
+
+For QDT, PAC file can be used if no environment variable for proxy are already defined (`HTTP_PROXY`,`HTTPS_PROXY`, `QDT_PROXY_HTTP`).
```

Comment:
Change priority:

1. QDT_PROXY_HTTP
2. System settings ---> pac or raw defined
3. HTTP_PROXY

Reply:
This was updated.